lib/cap/authors_poller.rb

```diff
@@ -212,19 +204,20 @@ def process_record_for_new_author(cap_profile_id, record)
           logger.warn "New author has authorship which will be skipped; Author.find_by(cap_profile_id: #{cap_profile_id})"
           @new_auth_with_contribs += 1
         end
-        queue_author_for_harvest(author)
+        queue_author_for_harvest(author,new_record: true)
         author.save!
         @new_author_count += 1
       end
 
       # Add author to job queue if it's harvestable and new/changed
       # @param [String] `skip_message` logs this message if it skips adding author to queue
-      def queue_author_for_harvest(author)
-        if (author.new_record? || author.changed?) && author.harvestable?
```

Comment:
NOTE:  author.new_record? will always returning false here, because the `process_record_for_new_author` method calls `Author.fetch_from_cap_and_create` which calls `Author.create!`.  This saves the record to the database.  Thus by the time we reach this method, we have no idea if the authors is new or not.  Hence passing this information in to the method.